src/lib/components/blog/copy-as-markdown.svelte

```diff
@@ -2,14 +2,13 @@
     import { getContext } from 'svelte';
     import { handleCopy } from '$lib/utils/copy';
     import { cn } from '$lib/utils/cn';
+    import { rawContent } from '$routes/docs/+layout.svelte';
 
     interface CopyAsMarkdownProps {
         class?: string;
     }
 
-    const content = getContext<string>('rawContent');
-
-    const { copy, copied } = handleCopy(content, 2000);
+    const { copy, copied } = handleCopy($rawContent!, 2000);
```

Comment:
null assertion seems dangerous because it's possible for `$rawContent` to be `null`